src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+
+/**
+ * The storage class to handle data saving
+ */
+public class Storage {
+    private final String filepath;
+
+    /**
+     * constructor for filepath
```

Comment:
The content of the javadoc is nice and clear! What can be improved would be the use of proper capitalisation and punctuation (period to end the sentence).